conf/command.conf

```diff
@@ -0,0 +1,83 @@
+# CustomCommand section must be defined before Latency monitor to support all commands
+#
+# NOTE: only support maximum 16 custom commands
+#
+# [...] must be specify
+# (...) optional. it has default value
+#
```

Comment:
in generally,
+ <...> means required
+ [...] means optional

Reply:
Changed as above.

Reply:
keep the unique style, see https://github.com/joyieldInc/predixy/blob/master/conf/auth.conf
+ <...> means required
+ [...] means optional